Header          header

# All of the people found
laser_touch_screen/PositionMeasurement[] people

# The co-occurrence matrix between people
float32[] cooccurrence